pyproject.toml

```
@@ -142,15 +142,44 @@ Discussions = "https://github.com/ansys/pyaedt/discussions"
 Releases = "https://github.com/ansys/pyaedt/releases"
 Changelog = "https://github.com/ansys/pyaedt/blob/main/doc/source/changelog.rst"
 
-[tool.black]
+[tool.ruff]
 line-length = 120
+exclude = [
+    "venv", 
+    "__init__.py", 
+    "doc/_build", 
+    "doc/source/examples"
+]
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
```

```suggestion
indent-style = "space"
docstring-code-format = true
```
The added configuration is something close to what `blacken-docs` was for `black`. I think we should use it and remove `blacken-docs`.
Also, we might want to use `docstring-code-line-length = "dynamic"` later on. Maybe in another PR?

---

As per our discussion, I will accept the suggested change.
The setting ``docstring-code-line-length = "dynamic"`` can be applied immediately as this is the default configuration for ``ruff`` when using ``docstring-code-format = true`` (see https://docs.astral.sh/ruff/settings/#format_docstring-code-line-length).
However, the pre-commit hook for ``blacken-docs`` will be kept, as it provides linting/formatting of Python code blocks in ``.rst`` files and should not create conflicts for docstring formatting with ``ruff``.